src/Components/FormRadioFieldSet.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { ValidationState } from '../Models/PatientFieldData';
+import ValidationMessage from './ValidationMessage';
+
+import './FormRadioFieldSet.scss';
+
+function FormRadioFieldSet({ children, labelText, property, isRequired, validationState }) {
+  return (
+    <fieldset
+      className={classNames('usa-fieldset form-radio-field-set', {
+        'form-radio-field-set--error': validationState === ValidationState.ERROR,
+        'form-radio-field-set--success': validationState === ValidationState.FIXED,
```

Comment:
Note that "modifier" classes in BEM should use `--` not `__` as the separator... renamed them...